lib/Core/Server/LanguageServer.php

```diff
@@ -239,13 +239,9 @@ private function handle(Connection $connection): Promise
                     continue;
                 }
 
-                \Amp\asyncCall(function () use ($serviceManager, $request, $container, $transmitter, $connection, $serverClient) {
+                \Amp\asyncCall(function () use ($request, $container, $transmitter, $connection) {
                     try {
-                        $response = yield $container->dispatch($request, [
-                            '_transmitter' => $transmitter,
-                            '_serverClient' => $serverClient,
-                            '_serviceManager' => $serviceManager,
-                        ]);
+                        $response = yield $container->dispatch($request, []);
```

Comment:
These "extra parameters" were worrysome from the start, and should always have been dependency injected - which is now possible AFAIK. Proably this will cause some issues...